OsmAnd/src/net/osmand/plus/plugins/srtm/SRTMPlugin.java

```diff
@@ -235,6 +249,7 @@ public void setEnabled(boolean enabled) {
 		if (mapRendererContext != null) {
 			mapRendererContext.updateElevationConfiguration();
 			mapRendererContext.recreateHeightmapProvider();
+			mapRendererContext.updateVerticalExaggerationScale();
```

Comment:
We can probably call here updateMapPresentationEnvironment to reduce copy-pasted code